api/controllers/PublicController.js

```diff
@@ -59,7 +59,7 @@ module.exports = {
           throw new Error('Email not recognised.')
         } else {
           randomString = ForgotPass.randomString()
-          return Members.update({ id: member.id }, { new_password: randomString })
+          return Members.update({ id: member.id }, { new_password: randomString, id: member.id })
```

Comment:
Yeah, sorry I forgot that Sails is like that. So annoying. Can we add a comment to this line?